lib/transformers/plugins/auth_validator_transformer.rb

```diff
@@ -4,20 +4,23 @@ module KrakendOpenAPI
   module Plugins
     # Transforms Auth Validator plugin definition
     class AuthValidatorTransformer
-      def transform_to_hash(roles:, config:)
-        raise(StandardError, 'Roles are empty') if roles.empty?
+      def transform_to_hash(config:, roles: [], scopes: [])
+        value = {
+          'alg': config['alg'],
+          'jwk_url': config['jwk_url'],
+          'cache': config['cache'],
+          'operation_debug': config['operation_debug'],
+          'roles_key_is_nested': config['roles_key_is_nested'],
+          'roles_key': config['roles_key'],
+          'scopes_key': config['scopes_key']
+        }.compact
```

Comment:
@denblackstache I think there's an argument for taking the entire config:

```suggestion
        value = config.dup
```

Reply:
@hawknewton Yeah, that makes sense.